### DISCUSSION TITLE
Baseline alignment for matrix's row

### DISCUSSION THREAD

**Discussion by ngochoan1011** [KaTeX/KaTeX#2487] [Q&A]
I have a matrix similar to this:

![image](https://user-images.githubusercontent.com/3677211/92144380-d493e000-ee40-11ea-88ab-d4d6632616bc.png)

How can I adjust the elements in the row of the current matrix to baseline alignment?

Currently, the row alignment for the matrix seems to be on the top. I have referred the https://github.com/KaTeX/KaTeX/issues/429 and https://github.com/KaTeX/KaTeX/issues/331 as which seems to be relating to my case but it doesn't work.

Thanks in advance.

**Comment by k4b7** [comment]
Can you post a screenshot of what you're trying to achieve?
